x-pack/plugin/ent-search/qa/rest/src/yamlRestTest/resources/rest-api-spec/test/entsearch/connector/sync_job/70_connector_sync_job_update_stats.yml

```diff
@@ -184,6 +184,36 @@ setup:
   - match: { indexed_document_volume: 1000 }
   - match: { last_seen: 2023-12-04T08:45:50.567149Z }
 
+---
+"Update the ingestion stats for a connector sync job - with optional metadata":
```

Comment:
I don't know if this is necessary, but should we also have a test for when metadata is in an unexpected format? (like a string or int).

Reply:
Yup, added tests. For str or int we would return 400 as a parsing exception (parser expects an obejct)